src/messages.ts

```diff
@@ -1,7 +1,7 @@
 import chalk from "chalk"
 import type { Preferences } from "./prompts"
-import { getUserPkgManager } from "./utils/getUserPkgManager"
 import { getVersion } from "./utils/getVersion"
+import { getUserPkgManager } from "./utils/getUserPkgManager"
```

Comment:
This change seems to be unnecessary, is there a reason you swapped the imports ?

Reply:
It turns out at first I thought I wouldn't need the package manager util but after some testing, I saw somewhere I could use it for better "safety".

Reply:
You should remove that change then ! 😁

Reply:
All should be well now